src/main/java/io/pinecone/proto/DeleteResponse.java

```diff
@@ -23,28 +23,28 @@ private DeleteResponse(com.google.protobuf.GeneratedMessageV3.Builder<?> builder
   private DeleteResponse() {
   }
 
-  @Override
+  @java.lang.Override
   @SuppressWarnings({"unused"})
-  protected Object newInstance(
+  protected java.lang.Object newInstance(
       UnusedPrivateParameter unused) {
     return new DeleteResponse();
   }
 
   public static final com.google.protobuf.Descriptors.Descriptor
       getDescriptor() {
-    return VectorServiceOuterClass.internal_static_DeleteResponse_descriptor;
+    return io.pinecone.proto.VectorServiceOuterClass.internal_static_DeleteResponse_descriptor;
   }
 
-  @Override
-  protected FieldAccessorTable
+  @java.lang.Override
```

Comment:
When I generate it locally, the Override is not replaced with java.lang.Override. I'm going to test it locally against serverless indexes and approve this PR soon.

Reply:
Also generating the code locally, there are no changes in the the following files: 
1. DeleteResponse
2. DeleteResponseOrBuilder
3. DescribeIndexStatsRequestOrBuilder
4. RequestUnion
5. RequestUnionOrBuilder
6. UpdateResponseOrBuilder

We can ignore files like DeleteResponseOrBuilder since you are using different protoc version but I'm not sure about the other changes.

Reply:
Okay I have tested with serverless indexes and things look good. Since Override is part of Java.lang, I'm not concerned why its expanding when you generate the code.